app/views/expense_categories/index.html.erb

```diff
@@ -0,0 +1,48 @@
+<p style="color: green"><%= notice %></p>
+<div class="category-info">
+  <h1>
+    <%= image_tag @category.icon %>
+    <%= @category.name %> Expenses
+  </h1>
+  <p><%= @category.description %> </p>
+</div>
+
+<div id="expense_categories">
+  <table class="table table-striped table-bordered table-hover">
+    <% if @expenses.empty? %>
+      <p>There are no expenses yet.</p>
+    <% else %>
+      <thead class="table-secondary">
+        <tr>
+          <th>Name</th>
+          <th>Amount</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @expenses.each do |expense| %>
+          <tr>
+            <td><%= expense.name %></td>
+            <td><span>$<%= expense.amount %></span></td>
+            <td><%= link_to "Show", expense_path(expense) %>
+            <%= link_to "Delete", expense_path(expense), data: {
+                    turbo_method: :delete,
+                    turbo_confirm: "Are you sure?"
+                  } %></td>
+          </tr>
+        <% end %>
+      </tbody>
```

Comment:
- [ ] The design must be follow. Please make sure to follow the design guideline what given.


### Given design
![Screenshot from 2022-10-28 05-04-17](https://user-images.githubusercontent.com/26767383/198417728-b8ffede3-b9d9-4727-aeaa-8e5cd84b21c2.png)


### Your design

![Screenshot from 2022-10-28 05-05-18](https://user-images.githubusercontent.com/26767383/198417748-296b307e-5f17-41f3-83a0-7e5e90a28116.png)

Reply:
Hello @Deepakdanger , please I have this page on my categories. The page that you have screenshotted is showing all transactions in each category. Please kindly check my categories page, and you will find what you are looking for. As you can see, there are different icons showing in each category just like the template.

By the way, the Behance design that was provided by microverse was only one-page styling. There are no proper designs for how all pages should look like. I have implemented it on my CATEGORIES page.

Thanks.